optimum/intel/utils/import_utils.py

```diff
@@ -73,8 +73,24 @@
 _openvino_version = "N/A"
 if _openvino_available:
     try:
+        import openvino_telemetry as tm

```

Comment:
should be here additional try-except for import telemetry only?

if user has openvino but not install openvino-telemetry it now can fail reporting that openvino is not available